src/test/rustdoc/nested-modules.rs

```diff
@@ -0,0 +1,17 @@
+#![crate_name = "foo"]
+
+mod a_module {
+  fn private_function() {
+      println!("called `a_module::private_function()`");
+  }
+
+  pub mod a_nested_module {
+    pub fn a_nested_public_function() {
+        println!("called `a_module::a_nested_module::a_nested_public_function()`");
+    }
```

Comment:
(Fill in the '...' with the relevant info.)

```suggestion
    }

    // @has 'foo/a_nested_module/index.html' '//a[@href="..."]' 'pub use ...'
    pub use a_nested_public_function as another_nested_public_function;
```